### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by SheldonCheng**
Various UI tweaks and fixes.

## Post and Post Detail
- Add proper date formatting
- Disable Connect button when viewing own Post
- Disabled button style
- "My Portfolio" link appears in Seeker post  if available

image

## Swap I'm Seeking / I'm Hiring button positions
"I'm Seeking" appears on the left and selected by default
image

## Other minor fixes
- Profile page title is now outside white box and centered
- PostDetail link (eg. My Portfolio) is no longer full width but follows length of text
- Fix toast popup not centered properly

### CODE REVIEW

File: ui/src/shared/components/PostComponents.tsx [hunk 1]
```diff
@@ -100,13 +100,17 @@ const PostLink = styled.a.attrs({ target: "_blank", rel: "noreferrer noopener" }
 `;
 
 export const PostLinks: React.FC<PostLinksProps> = ({ data }) => {
+  console.log(data);
```
Comment (jacobtyq) [inline ui/src/shared/components/PostComponents.tsx]: Can remove log

File: ui/src/shared/components/PostComponents.tsx [hunk 2]
```diff
@@ -97,12 +100,17 @@ const PostLink = styled.a.attrs({ target: "_blank", rel: "noreferrer noopener" }
...
-  );
+  console.log(data);
+  /* eslint-disable @typescript-eslint/camelcase */
+  const { job_link, company_link, website } = data;
+  if (job_link || company_link || website) {
+    return (
+      <PostLinksContainer>
+        {job_link && <PostLink href={job_link}>Job Details</PostLink>}
+        {company_link && <PostLink href={company_link}>Company Info</PostLink>}
+        {website && <PostLink href={website}>My Portfolio</PostLink>}
```
Comment (jacobtyq) [inline ui/src/shared/components/PostComponents.tsx:110]: Similar to this bug @ng95junwei mentioned, if we have a URL without the http/https protocol (e.g. www.getajob.com), the URL becomes a relative link.

I think we can either get users to fill in a correct URL (via validation on Front End) or accept links and convert it ourselves either on Back End or when displaying it (conversion to a proper URL)

@subhransu what do you think? 

Screenshot 2020-10-24 at 12 39 31 AM
^ See bottom left

**Review by jacobtyq — APPROVED** [review]
LGTM, left some comments

File: ui/src/shared/components/PostComponents.tsx (hunk 2 above)
Comment (SheldonCheng) [reply ui/src/shared/components/PostComponents.tsx:110]: Let's carry on this discussion separately, not part of this PR's scope.

**Pull request merged**